plugins/polio/js/src/components/CreateEditDialog.js

```diff
@@ -169,10 +169,6 @@ const CreateEditDialog = ({ isOpen, onClose, campaignId }) => {
             {
                 title: formatMessage(MESSAGES.preparedness),
                 form: PreparednessForm,
-                hasTabError: compareArraysValues(
-                    preparednessFormFields,
-                    formik.errors,
-                ),
```

Comment:
There's an unused import to delete related to this

Reply:
I've removed it. thank you